third-party/jemalloc/README

```diff
@@ -0,0 +1,11 @@
+==========================
+jemalloc README for Chapel
+==========================
+
+This copy of jemalloc is being released with Chapel for convenience and
+uses a snapshot from:
```

Comment:
We should somehow identify the snapshot, either by timestamp or, better yet, the SHA of the last commit in the jemalloc repo.